apps/server/src/controllers/userControllers/getUserProfileDetails.ts

```diff
@@ -0,0 +1,33 @@
+import { Response } from 'express'
+import { RequestExt } from 'middleware/authorization/authentication'
+import { HTTP_STATUS_CODE } from '../../constants'
+
+export const userProfileDetails = async (req: RequestExt, res: Response) => {
+  try {
+    const { _user: user } = req.body
+    res.status(HTTP_STATUS_CODE.SUCCESS).json({
+      message: 'Fetched user profile details succesfully',
+      user: {
+        firstName: user.firstName,
+        lastName: user.lastName,
+        email: user.email,
+        phone: user.phone,
+        gender: user.gender,
+        occupation: user.occupation,
+        date_of_birth: user.date_of_birth,
+        bvn: user.bvn,
+        address: user.address,
+        identification_type: user.identification_type,
+        identification_number: user.identification_number,
+        identification_doc: user.identification_doc,
+        proof_of_address_doc: user.proof_of_address_doc,
+        isVerified: user.isVerified,
```

Comment:
I'm wondering if its very necessary to return this entire object??

Reply:
Well, what is needed are the profile details, which are also the user object. Shouldn't I return the entire object??